src/Framework/Menus/CustomBoard.cs

```diff
@@ -116,19 +118,21 @@ public override void snapToDefaultClickableComponent()
         public override void gameWindowSizeChanged(Rectangle oldBounds, Rectangle newBounds)
         {
             base.gameWindowSizeChanged(oldBounds, newBounds);
+            Game1.activeClickableMenu = new CustomBoard(this._boardName, this._billboardTexture);
```

Comment:
Why?

Reply:
Same reasoning as below. Reopens the menu on window resize such that the buttons are placed correctly.